src/NuGet.Core/NuGet.Build.Tasks.Pack/DefaultSymbolsPackageFormatChangingWarningTask.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using Microsoft.Build.Utilities;
+using NuGet.Common;
+
+namespace NuGet.Build.Tasks.Pack
+{
+    public class DefaultSymbolsPackageFormatChangingWarningTask : Task
+    {
+        public ILogger Logger => new MSBuildLogger(Log);
+
+        public override bool Execute()
+        {
+            Logger.Log(LogMessage.CreateWarning(NuGetLogCode.NU5132,
```

Comment:
Will it create a warning in each nuget.exe/dotnet.exe/msbuild pack operation?

Is it better to show the warn only when you're creating a symbols package?

Reply:
I thought this was a IPackageRule. Ignore my comment.